plugins/automox/plugin.spec.yaml

```diff
@@ -598,180 +598,252 @@ types:
+  solution_vulnerability:
+    id:
+      title: Vulneability ID
+      description: Identifier of the vulnerability, typically a CVE
+      type: string
+      required: true
+    title:
+      title: Title
+      description: Title of the vulnerability
       type: string
       required: false
-    updated_at:
-      title: Updated At
-      description: The datetime the task was updated
+    summary:
+      title: Summary
+      description: Summary of the vulnerability
       type: string
       required: false
-    completed_at:
-      title: Completed At
-      description: The datetime the task was completed
+    severity:
+      title: Severity
+      description: Severity of the vulnerability
       type: string
       required: false
-    cves:
-      title: CVEs
-      description: List of CVEs associated with task
-      type: "[]string"
+  solution:
+    id:
+      title: Solution ID
+      description: Identifier of solution
+      type: integer
+      required: true
+      example: 1234
+    solution_type:
+      title: Solution Type
+      description: Type of solution
+      type: string
       required: false
-    report_source:
-      title: Report Source
-      description: The source of the vulnerability report
+      example: "rapid7-solution"
+    remediation_type:
+      title: Remediation Type
+      description: Type of remediation
       type: string
       required: false
+      example: "patch-with-worklet"
+    solution_details:
+      title: Solution Details
+      description: Details of the solution. This can include package information and other details depending on the solution type
+      type: object
+      required: false
+      example:
+        solution_id: "office-click-to-run-upgrade-latest"
+        solution_type: "workaround"
+        solution_summary: "Upgrade to the latest version of Microsoft Office"
+        solution_fix: "Install Office Click-To-Run updates through any installed Office application. Go to File &gt; Account (or Office Account if you opened Outlook). Under Product Information, choose Update Options &gt; Update Now."
+    device_ids:
+      title: Device IDs
+      description: List of device identifiers associated with the solution. This is a helper field to make executing actions easier. It does not exist on the Automox API.
+      type: "[]integer"
+      required: false
+      example: [ 1234, 5678 ]
+    devices:
+      title: Devices
+      description: List of devices associated with the solution
+      type: "[]solution_device"
+      required: false
+      example:
+        - id: 1234
+          name: "device-1"
+          custom_name: "custom-name"
+          status: "in_progress"
+          deleted: false
+          ip_addrs_private:
+            - 10.0.0.1
+    vulnerabilities:
+      title: Vulnerabilities
+      description: List of vulnerabilities associated with the solution
+      type: "[]solution_vulnerability"
+      required: false
+      example:
```

Comment:
I don't believe these examples will be used. They shouldn't upset anything if left in however.